assets/training/model_evaluation/components/compute_metrics/spec.yaml

```diff
@@ -77,7 +77,7 @@ outputs:
 
 is_deterministic: True
 code: ../../src
-environment: azureml://registries/azureml/environments/model-evaluation/versions/32
+environment: azureml://registries/azureml/environments/model-evaluation/versions/36
```

Comment:
can you please check if we can update the environment to latest tag, so that we might need to update this manually?